perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -8,10 +8,13 @@
 #pragma comment (lib, "sim86_shared_debug.lib")
 #define ArrayCount(Array) (sizeof(Array) / sizeof((Array)[0]))
 
-static constexpr int RegisterCount = 15;
-static constexpr int IPRegister = 13;
-static constexpr int CXRegister = 3;
-static constexpr u16 InvalidValue = 0xFFFF;
+static constexpr size_t MEGABYTE = 1024 * 1024;
+static constexpr int REGISTER_COUNT = 15;
+static constexpr int IP_REGISTER = 13;
+static constexpr int CX_REGISTER = 3;
+static constexpr u16 INVALID_VALUE = 0xFFFF;
+
+static u8 Memory[MEGABYTE];
```

Comment:
I would move this lower in the file, below all the constant stuff. It's just my personal preference, so don't feel like you have to do it. My reason is that I just like to group all type definitions together, all constants together, and all globals together. Easier to find them this way.

Reply:
My reasoning for this one was to group it with other arrays, but I didn't consider that they were constant arrays.
This makes more sense
564965a8f855c6eecf2fa6467cc688831b42f68b